docs/CreateAKSClusterForScalarDB.md

```diff
@@ -0,0 +1,156 @@
+# Create an AKS cluster for ScalarDB Server
+
+This document explains how to create an Azure Kubernetes Service (AKS) cluster for ScalarDB Server deployment. For details on how to deploy ScalarDB Server on an AKS cluster, see [Deploy ScalarDB Server on AKS](./ManualDeploymentGuideScalarDBServerOnAKS.md).
```

Comment:
```suggestion
This document explains the requirements and recommendations for creating an Azure Kubernetes Service (AKS) cluster for ScalarDB Server deployment. For details on how to deploy ScalarDB Server on an AKS cluster, see [Deploy ScalarDB Server on AKS](./ManualDeploymentGuideScalarDBServerOnAKS.md).
```
As mentioned above, users don't create a cluster by following the document, so it might be misleading to say `This document explains how to create an Azure Kubernetes Service`.

Reply:
I also updated this part in 2a63f33145871a88697194f734005123c3a551b3.